src/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 cmake_minimum_required(VERSION 3.3)
 
 if(NOT DEFINED ${medInria_VERSION})
-    set(medInria_VERSION 3.0.1)
+    set(medInria_VERSION 3.1.0)
```

Comment:
this is going to automatically change the version number asked in medHomepageArea line 161, and displayed on the Homepage->About